### ISSUE TITLE
Magic Link: Chromebook won't let you log in

### ISSUE THREAD

**Issue by rossanafmenezes** [wordpress-mobile/WordPress-Android#15000]
### Expected behavior

I believe the Chromebook is in some kind of limbo here since it's not a mobile device but uses Android and you download the Android app from the Google Store. The log-in using the Magic link option is there, so it's safe to assume the users would think you can log into the Chromebook app using that option. 

### Actual behavior

But it won't let you log into the app. Instead, it will show this message:

![photo_2021-07-06 11 42 28](https://user-images.githubusercontent.com/30724184/124756032-09a60e80-defa-11eb-95b3-98a9b930471e.jpeg)

### Steps to reproduce the behavior

Install the WordPress App on a Chromebook
Click on the WordPress.com login option
Enter your email
Select the Magic Link option

##### Tested on [device], Android [version], WPAndroid [version]

4117628-zen
Pixelbook
Google Chrome OS 91.0.4472.102 (official build) (64 bit)

Not sure about the user's device, will ask for more info.

**Comment by thehenrybyrd** [comment]
@rossanafmenezes thanks for reporting!
It sounds like the user is trying to use the WP Android app on their Chromebook, rather than on a more traditional mobile device, is that correct?

**Comment by thehenrybyrd** [comment]
It looks like we're identifying the Chromebook as a laptop rather than an Android device. 
However, based on a quick conversation with @rossanafmenezes, the WP Android app can be downloaded from the Google Store on a Chromebook (along with most other Android apps) and is expected to act like a mobile app.

@ashiagr or @zwarm can one of you take a look?

**Comment by rossanafmenezes** [comment]
Yeah, that's right. I'm available to help with testing if needed since I have a Chromebook here.

**Comment by ashiagr** [comment]
Thanks, @rossanafmenezes! Can you possibly DM (on Slack) the login magic link that you received in the email?

We'll have to see how [device detection](https://opengrok.a8c.com/source/xref/trunk/wp-content/mu-plugins/login-endpoint/magic-login.php?r=223213&mo=5635&fi=183#135-153) works for the Chromebook at the API level.
